benchmark/src/main.rs

```diff
@@ -89,6 +89,11 @@ struct Args {
     #[clap(long, env)]
     frequency_penalty: Option<f32>,
 
+    /// Generation parameter in case you want to specifically test/debug particular
+    /// decoding strategies, for full doc refer to the `text-generation-server`
+    #[clap(long, env)]
+    no_repeat_ngram_size: Option<u32>,
```

Comment:
this comment may need to be updated

Reply:
Should we update all of them of in that case 🤔

Reply:
oh haha good point, I didnt realize all the args have a similar comment